src/main/java/seedu/address/model/meeting/MeetingTime.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.model.meeting;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents the meeting time for a meeting.
+ */
+public class MeetingTime {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Meeting time must be in the following format: YYYY-MM-dd HH:mm\n Example: 2022-05-25 23:59";
```

Comment:
Perhaps in the future we can support multiple date-time formats in ParserUtil but for now I think this is nice